File: java/empleados/Repartidor.java
public class Repartidor extends Empleado {
    private String zona;

    public Repartidor(String zona,String nombre,int edad,double salario){
        super(nombre,edad);
        if(edad < 25 && zona.toLowerCase() == "zona 3"){
            this.setSalario(salario + this.PLUS); 
        }else{
            this.setSalario(salario); 
        }
        
        this.zona = zona;
    }

    @Override 
    public String toString(){
        return 
         " Zona: " + this.zona +
          "\n Nombre: " + this.getNombre() +
          "\n Edad: " + this.getEdad() +
          "\n Salario: " + this.getSalario();
    }

}

File: dart/Empleados/Repartidor.dart
import 'Empleado.dart';

class Repartidor extends Empleado{
  String _zona = "";
  Repartidor(nombre,edad,salario,String zona):super(nombre,edad,salario){
    if(edad>25 && zona.toLowerCase() == "zona 3"){
      this.setSalario = salario + this.PLUS;
    }
   this._zona = zona;
  }

  @override 
  String toString(){
    return super.toString() + "\nZona: " + this._zona;
  }

}